adr/0015-activity-logging.md

```diff
@@ -0,0 +1,54 @@
+# 15. Activity Logging
+
+Date: 2024-10-02
+
+## Status
+
+Accepted
+
+## Context
+
+Trento feature-set is growing and not limited to only reading operations but expanding to applying state changes of different natures:
+- tagging/untagging resources
+- applying trento settings (api key, SUMA, etc...)
+- user/profile management operations (creation, modification, deletion)
+- operate on the monitred infrastructure (apply patches/upgrade packages via SUMA integration, apply remediations for failing checks)
```

Comment:
`apply patches/upgrade packages via SUMA integration`. 
We don't know how the patching will be done.
This looks a lot to presume.